packages/ui/src/components/molecules/Tag/styles.scss

```diff
@@ -0,0 +1,99 @@
+[data-fs-tag] {
+  // --------------------------------------------------------
+  // Design Tokens for Tag
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-tag-text-color                     : var(--fs-color-text);
+
+  // Icon
+  --fs-tag-icon-size                      : var(--fs-spacing-3);
+  --fs-tag-icon-stroke-width              : var(--fs-spacing-4);
+
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  [data-fs-badge-wrapper] {
+    position: relative;
+    display: flex;
+    align-items: center;
+    padding-right: var(--fs-spacing-6);
+    line-height: initial;
+  }
+
+  [data-fs-tag-icon-button] {
+    position: absolute;
+    z-index: 1;
+    right: calc(-1 * var(--fs-spacing-1));
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--fs-control-tap-size);
+    height: var(--fs-control-tap-size);
```

Comment:
This `--fs-control-tap-size` is rendering `54px`, shouldn't it be `48px`?

Reply:
I noticed that this is a general problem, trying to fix it :)